packages/l/link-styler.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "link-styler",
+  "filename": "link-styler.js",
+  "description": "JavaScript package that simplifies the task of identifying links within text and applying custom styling by automatically replacing them with tags.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/sheikhoo/link-styler"
+  },
+  "license": "MIT",
+  "homepage": "https://link-styler.js.org",
+  "keywords": [
+    "autolink",
+    "hashtag",
+    "mention"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/sheikhoo/link-styler",
```

Comment:
Our auto-update mechanism requires that either there exists a source repository with versions tagged that contain the required files for the CDN, or a package released on NPM that contains the files needed for the CDN.

It looks like you have a versioned NPM package, so switching to using that might be best?

Reply:
Thank you, I changed git to npm